projects/kit/components/input-date/input-date.component.ts

```diff
@@ -123,9 +130,14 @@ export class TuiInputDateComponent
         private readonly mobileCalendar: Type<any> | null,
         @Inject(TUI_TEXTFIELD_SIZE)
         private readonly textfieldSize: TuiTextfieldSizeDirective,
-        @Inject(TUI_DATE_FILLER) readonly filler: string,
+        @Inject(TUI_DATE_FORMAT) readonly dateFormat: TuiDateMode,
+        @Inject(TUI_DATE_TEXTS)
+        readonly dateTexts$: Observable<Record<TuiDateMode, string>>,
     ) {
         super(control, changeDetectorRef);
+        this.dateTexts$
```

Comment:
You can rewrite all of those this way:
```
readonly filler$ = this.dateTexts$.pipe(
  map(texts => texts[this.dateFormat]),
)
```
And use with `async` pipe. Use `ngIf` or `tuiLet` to declare the result if you need it in multiple places.

Reply:
`tuiCreateAutoCorrectedDatePipe` requires from this component to have sync method `filler`

Reply:
Then it could be:
```
textMask$ = this.filler$.pipe(
  ...
)
```

Reply:
i created `filler$ ` and made subscription via `tuiLet`